test/e2e/simple.sh

```diff
@@ -47,5 +47,18 @@ RELEASE_IMAGE_LATEST="$( curl -s -H "Accept: application/json"  "https://api.ope
 export RELEASE_IMAGE_LATEST
 os::cmd::expect_success "ci-operator --secret-dir ${PULL_SECRET_DIR} --target [release:latest] --config ${suite_dir}/dynamic-releases.yaml"
 unset RELEASE_IMAGE_LATEST
+os::test::junit::declare_suite_end
 
+os::test::junit::declare_suite_start "e2e/simple/optional-operator"
+if [[ -z "${PULL_BASE_SHA:-}" ]]; then
+  os::log::fatal "\$PULL_BASE_SHA must be set for this test"
+fi
+if [[ -z "${PULL_NUMBER:-}" ]]; then
+  os::log::fatal "\$PULL_NUMBER must be set for this test"
+fi
+if [[ -z "${PULL_PULL_SHA:-}" ]]; then
+  os::log::fatal "\$PULL_PULL_SHA must be set for this test"
+fi
+export JOB_SPEC='{"type":"postsubmit","job":"branch-ci-openshift-ci-tools-master-ci-operator-e2e","buildid":"0","prowjobid":"uuid","refs":{"org":"openshift","repo":"ci-tools","base_ref":"master","base_sha":"'"${PULL_BASE_SHA}"'","pulls":[{"number":'"${PULL_NUMBER}"',"sha":"'"${PULL_PULL_SHA}"'","author":"AlexNPavel"}]}}'
+os::cmd::expect_success "ci-operator --image-import-pull-secret ${PULL_SECRET_DIR}/.dockerconfigjson --target [images] --config ${suite_dir}/optional-operators.yaml"
```

Comment:
Could we somehow write a simple, lightweight `tests` step in this config that either runs in the index or uses my PR to depend on the index image and does something trivial to validate that the index is workable?